.github/workflows/tempSense_sky130hd.yml

```diff
@@ -39,7 +39,7 @@ jobs:
               cp ./.github/scripts/parse_rpt.py ./openfasoc/generators/temp-sense-gen/. &&\
               pip3 install -r requirements.txt &&\
               cd ./openfasoc/generators/temp-sense-gen &&\
-              make sky130hd_temp &&\
+              make sky130hd_temp_full &&\
```

Comment:
I suggest keeping flow till drc/lvs checks and the full flow till simulation checks separately. It would be easy to understand which part is failing instead of digging into the log files.

Reply:
Do you mean that I should have one job with `make ..` and another job with `make ..full`?